static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import { imrWorldwide } from './imr-worldwide';
 
-const { shouldRun, url, onLoad } = imrWorldwide;
+const { shouldRun, url } = imrWorldwide;
+const onLoad: any = imrWorldwide.onLoad;
```

Comment:
I'm torn with which pattern to follow. This removes the need to `$FlowFixMe`, however I would love to avoid using `any`. Also, I would like the Flow rulings in our tests to maybe be less strict, so tests can be as similar to the running code as possible... Third-party-tags also assumes onLoad will be a function.

Should we have to customise our import pattern just to make Flow happy?